python/tk_multi_loader/dialog.py

```diff
@@ -146,7 +148,8 @@ def __init__(self, action_manager, parent=None):
         self._publish_model.cache_loaded.connect(self._on_publish_content_change)
         self._publish_model.data_refreshed.connect(self._on_publish_content_change)
         self._publish_proxy_model.filter_changed.connect(self._on_publish_content_change)
-
+        self._publish_model.data_refreshed.connect(self._status_filter_change)
```

Comment:
Maybe rename "_status_filter_change" as "_on_status_filter_change" to stay consistent with "_on_publish_content_change" ?